CHANGELOG.md

```diff
@@ -14,6 +12,12 @@
 
 ### Dependency updates
 
+## [16.0.5] - 2022-09-16
+
+### Added
+
+- `Toggle`: `tooltip` and `tooltipPosition` props. ([@qubis741](https://github.com/qubis741)) in [#2355](https://github.com/teamleadercrm/ui/pull/2355))
```

Comment:
Sorry should've already checked this in my last review 🙈 
This is a new feature, so should be `16.1.0` right?

Reply:
oh yeah, true, thanks